src/connection/index.ts

```diff
@@ -70,28 +70,28 @@ export const gnosisSafeConnection: Connection = {
   shouldDisplay: () => false,
 }
 
-const [web3WalletConnect, web3WalletConnectHooks] = initializeConnector<WalletConnectPopup>(
-  (actions) => new WalletConnectPopup({ actions, onError })
+const [web3WalletConnect, web3WalletConnectHooks] = initializeConnector<WalletConnectV1WithAnalytics>(
+  (actions) => new WalletConnectV1WithAnalytics({ actions, onError })
 )
-export const walletConnectConnection: Connection = {
-  getName: () => 'WalletConnect',
+export const walletConnectV1Connection: Connection = {
+  getName: () => 'WalletConnectV1',
   connector: web3WalletConnect,
   hooks: web3WalletConnectHooks,
   type: ConnectionType.WALLET_CONNECT,
   getIcon: () => WALLET_CONNECT_ICON,
-  shouldDisplay: () => !getIsInjectedMobileBrowser(),
+  shouldDisplay: () => false,
 }
 
-const [web3WalletConnectV2, web3WalletConnectV2Hooks] = initializeConnector<WalletConnectV2Popup>(
-  (actions) => new WalletConnectV2Popup({ actions, onError })
+const [web3WalletConnectV2, web3WalletConnectV2Hooks] = initializeConnector<WalletConnectV2WithAnalytics>(
+  (actions) => new WalletConnectV2WithAnalytics({ actions, onError })
 )
-export const walletConnectV2Connection: Connection = {
-  getName: () => 'WalletConnectV2',
+export const walletConnectConnection: Connection = {
```

Comment:
I think this should remain named `walletConnectV2Connection` to help disambiguate, as long as both V1 and V2 exist in the codebase.

Reply:
In general, I don't think anything should be named `WalletConnect` while both V1 and V2 exist in the codebase - they should all be suffixed.

Reply:
that seems reasonable, but I'm leaving the one on line 89 because that's the string that gets rendered into the UI.

Reply:
yes agree, rendered UI is different than code